### PR TITLE
Deprecate hook integration

### PR DESCRIPTION
**Pull request by springstan**
**Description:**
- see home-assistant/home-assistant#31046

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#31046

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
The logo, redirects file and links from other pages to this page, need to be cleaned up as well.

**Comment by springstan** [comment]
How can I find all links from other pages to this page?

**Comment by frenck** [comment]
Do a search for `/integrations/hook`?

**Comment by springstan** [comment]
Thanks for the tip @frenck, removed all of them.

File: source/_posts/2016-12-03-remote-websockets-sonarr.markdown [hunk 1]
```diff
@@ -212,7 +212,6 @@ Experiencing issues introduced by this release? Please report them in our [issue
 [gpslogger]: /integrations/gpslogger
 [harmony]: /integrations/harmony
 [homematic]: /integrations/homematic/
-[hook]: /integrations/hook
```
Comment (frenck) [inline source/_posts/2016-12-03-remote-websockets-sonarr.markdown:215]: Just removing the Markdown reference is not enough, you have to make sure the link that refers to this is also removed.

E.g., if there is written on that page: `Nice little [Hook][hook] integration` it becomes -> `Nice little Hook integration`.

Comment (springstan) [reply source/_posts/2016-12-03-remote-websockets-sonarr.markdown:215]: Hope I got all of them now 😅

File: source/_posts/2016-12-03-remote-websockets-sonarr.markdown [hunk 2]
```diff
@@ -36,14 +36,13 @@ The support for multiple hosts is a result of allowing mixed configurations with
...
 
-## All changes 
+## All changes
 
 - New services and improved device support for [HomeMatic][homematic] ([@pvizeli], [@danielperna84])
 - Device tracker: New support for [GPSLogger][gpslogger] ([@dainok])
 - Sensor: Support for [Sonarr][sonarr] ([@hborawski])
 - Sensor: [World Air Quality Index][waqi] sensor ([@valentinalexeev], [@fabaff])
 - Sensor: Support for [Dutch Smart Meter Requirements][dsmr] ([@aequitas])
-- Switch: [Hook][hook] support by hooksmarthome.com ([@dasos])
```
Comment (balloob) [inline source/_posts/2016-12-03-remote-websockets-sonarr.markdown:46]: I wonder if we should keep the changelog line, but just remove the link. What do you think Frenck?

Comment (frenck) [reply source/_posts/2016-12-03-remote-websockets-sonarr.markdown:46]: Yes, as I mentioned here: <https://github.com/home-assistant/home-assistant.io/pull/11832#discussion_r369098417>

Comment (springstan) [reply source/_posts/2016-12-03-remote-websockets-sonarr.markdown:46]: Good point ✌ done with https://github.com/home-assistant/home-assistant.io/pull/11832/commits/408340b8c9d676dda28b2c48b3efb26f7f428085

**Review by frenck — APPROVED** [review]

**Pull request merged**